pkg/master/master.go

```diff
@@ -251,6 +253,7 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		batchrest.RESTStorageProvider{},
 		certificatesrest.RESTStorageProvider{},
 		extensionsrest.RESTStorageProvider{ResourceInterface: thirdparty.NewThirdPartyResourceServer(s, s.DiscoveryGroupManager, c.StorageFactory)},
+		networkingrest.RESTStorageProvider{},
```

Comment:
Just checked again; they come back as `networking.k8s.io/v1`. That's also true if I add an AddCohabitatingResources() call, regardless of which order I list the two resource types there... So maybe something else is wrong somewhere else? (I don't really understand all the reststorage stuff, and was relying on reviewers to tell me if I'd done anything wrong. See https://github.com/kubernetes/kubernetes/pull/39164#issuecomment-295334789)